open-company/open-company-web
# No boards

#### NB: merge into #259 

To fix:
> If you delete all boards, you have a blank page and are stuck. Instead, let’s show the empty board illustration with the words: “You don’t have any boards yet. Add one?” (the link takes you to board creation)

From the [QA doc](https://docs.google.com/document/d/1X3j869Y8n94zfqaGVhrWf8VOlr3qEa_4a1fxtIjH6fs/edit#).
Since in `change-service` and in `boards-preload` there are substantial changes to how and when the org and board data are loaded this was branched out of `staging` that contains both of them and the current mainline.

To test:
 - create a new org from /create-org (the org avatar will probably be broken but don't worry)
- remove the Welcome board from the newly created org
- do you see the new empty org component? good
- try to create a new board using the _Add one?_ button
